src/Sylius/Bundle/AdminBundle/Resources/config/grids/shipment.yml

```
@@ -58,17 +58,24 @@ sylius_grid:
                             sylius.ui.ready: ready
                             sylius.ui.shipped: shipped
                 channel:
-                    type: entity
-                    label: sylius.ui.channel
-                    options:
-                        fields: [order.channel]
+                    type: ux_autocomplete
+                    label: sylius.ui.shipping_method
```

```suggestion
                    label: sylius.ui.channel
```
but if I see correctly, this label is ignored?

---

Not anymore